tests/codegen/float/f16-f128-inline.rs

```diff
@@ -0,0 +1,67 @@
+//! Ensure that functions using `f16` and `f128` are always inlined
+
+#![crate_type = "lib"]
+#![feature(f128)]
+#![feature(f16)]
+
+/* Check for direct arguments and return types */
+
+pub fn f16_arg(_a: f16) {
+    // CHECK-NOT: f16_arg
+    todo!()
```

Comment:
Are you sure the automatic inlining heuristics wouldn't anyway already mark these functions as inlineable? Might be worth setting cross_crate_inline_threshold to 0 to be sure that doesn't mask the test.

Reply:
I added a revision with `-Zcross-crate-inline-threshold=never -Zmir-opt-level=0 -Cno-prepopulate-passes`, is that reasonable?

Reply:
No, you also need `-Copt-level=0`.

Reply:
Oh yeah of course. Done, tests still pass.

Reply:
`-Cno-prepopulate-passes` just disables LLVM optimizations, rustc changes its behavior in a few places based on the value of `opt-level`.